CRM/Contact/BAO/Group.php

```diff
@@ -439,8 +451,10 @@ public static function create(&$params) {
 
       if (!empty($params['parents'])) {
         foreach ($params['parents'] as $parentId => $dnc) {
-          if ($parentId && !CRM_Contact_BAO_GroupNesting::isParentChild($parentId, $group->id)) {
-            CRM_Contact_BAO_GroupNesting::add($parentId, $group->id);
+          if (CRM_Utils_Type::validate($parentId, 'Integer')) {
```

Comment:
I guess we can get away without it, i just went for getting every possible local and validate it but i'm happy to remove it if you think its not needed